modules/client-configuration/pages/contact-methods-saltssh-tunnel.adoc

```diff
@@ -26,6 +26,12 @@ In other words, access is only possible if the session is initiated by the serve
 All package managing operations on the client can be performed from the server side only.
 ====
 
+
+[IMPORTANT]
+====
+Use [literal]``mgrctl term`` before running steps inside the server container.
+====
```

Comment:
The key is to say which steps need to run inside of the container and which not. This sentence expect that the customer know that need to be executed where.

Reply:
yes, we still must find a good way to communicate it.